docs/autonomy.md

```diff
@@ -75,7 +75,7 @@ These keys can be used for local deployments if you're using the default hardhat
 
 ```bash
 # create a docker deployment
-$ autonomy deploy build deployment valory/oracle_hardhat:bafybeibhv2ziivbnj3sgfzjjtvqbsvt3fvra4texc3lx4snqxo6lbmq2le keys.json
+$ autonomy deploy build deployment valory/oracle_hardhat:bafybeigw2hpuir72fca6oxfvh4kmy3rrllzivchw63tdi5ylkzvvssybvm keys.json
```

Comment:
@dvilelaf @jmoreira-valory Can you guys check if this works?

Reply:
It's working on my side

Reply:
Why has oracle_hardhat changed its hash here?

Reply:
Oh I thought this was hello world service